introduction.md

```diff
@@ -0,0 +1 @@
+Hi, this is Michael. I come from China and I am a college student.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.